### ISSUE TITLE
Export no longer works on mapshaper.org

### ISSUE THREAD

**Issue by gamecubate** [mbloch/mapshaper#216]
As of this morning, I am unable to export anything from mapshaper.org. Error is always the same: ```Export failed for an unknown reason.```

Browsers: Opera, Chrome.

The problem persists for both geojson and shapefile imports and exports.

JS console reports:
```
mapshaper-gui.js:2261 TypeError: "list" argument must be an Array of Buffers
    at Function.concat (modules.js:8000)
    at Object.GeoJSON.joinOutputBuffers (mapshaper.js:12212)
    at Object.GeoJSON.formatGeoJSON (mapshaper.js:12199)
    at Object.internal.exportDatasetAsGeoJSON (mapshaper.js:12187)
    at mapshaper.js:12053
    at Array.map (<anonymous>)
    at internal.exportGeoJSON (mapshaper.js:12048)
    at Object.internal.exportFileContent (mapshaper.js:14751)
    at mapshaper.js:14699
    at Array.reduce (<anonymous>)
```

**Comment by mbloch** [comment]
I've reverted the website to the previous version... looking in to the cause of the problem.

**Comment by gamecubate** [comment]
Works like a charm, again. Thanks a lot! (and perhaps the fastest response time I have ever witnessed for open source).

**Comment by mbloch** [comment]
I just published v0.4.51, which should fix this. (The bug was caused by accidentally trying to concatenate Uint8Array objects using the Buffer.concat() function).

Thanks for reporting this bug! :)
